### PR TITLE
Cleanup unused variables from color themes

### PR DESCRIPTION
**Pull request by SuperTux88**

### CODE REVIEW

File: app/assets/stylesheets/color_themes/dark_green/_style.scss [hunk 1]
```diff
@@ -3,16 +3,11 @@ $brand-primary: #009900;
...
-$main-color-light: lighten($brand-primary, 10%);
 $main-color-dark: darken($brand-primary, 20%);
 
 /* Bootstrap Variables */
 $btn-primary-bg: $brand-primary;
 $link-color: $brand-primary;
-
-/* Custom Variables */
-$header-background-color: $main-color-dark;
 $navbar-inverse-bg: $main-color-dark;
```
Comment (diaspora-code-review) [inline app/assets/stylesheets/color_themes/dark_green/_style.scss]: Prefer single quoted strings

File: app/assets/stylesheets/color_themes/egyptian_blue/_style.scss [hunk 2]
```diff
@@ -3,16 +3,11 @@ $brand-primary: #1034A6;
...
-$main-color-light: lighten($brand-primary, 10%);
 $main-color-dark: darken($brand-primary, 15%);
 
 /* Bootstrap Variables */
 $btn-primary-bg: $brand-primary;
 $link-color: $brand-primary;
-
-/* Custom Variables */
-$header-background-color: $main-color-dark;
 $navbar-inverse-bg: $main-color-dark;
```
Comment (diaspora-code-review) [inline app/assets/stylesheets/color_themes/egyptian_blue/_style.scss]: Prefer single quoted strings

File: app/assets/stylesheets/color_themes/magenta/_style.scss [hunk 3]
```diff
@@ -3,16 +3,11 @@ $brand-primary: #FF00FF;
...
-$main-color-light: lighten($brand-primary, 10%);
 $main-color-dark: darken($brand-primary, 30%);
 
 /* Bootstrap Variables */
 $btn-primary-bg: darken($brand-primary, 5%);
 $link-color: $brand-primary;
-
-/* Custom Variables */
-$header-background-color: $main-color-dark;
 $navbar-inverse-bg: $main-color-dark;
```
Comment (diaspora-code-review) [inline app/assets/stylesheets/color_themes/magenta/_style.scss]: Prefer single quoted strings

File: app/assets/stylesheets/color_themes/egyptian_blue/_style.scss (hunk 2 above)
Comment (denschub) [inline app/assets/stylesheets/color_themes/egyptian_blue/_style.scss]: 💻: 🎉 => 🍺 => 🍻 => 💩

**Comment by denschub** [comment]
Merged as 89a6379eddc744491868807fdb32804789fc998f, danke.

**Pull request closed without merging**